scripts/release_ppa.sh

```diff
@@ -117,11 +117,10 @@ version=$("$(dirname "$0")/get_version.sh")
 commithash=$(git rev-parse --short=8 HEAD)
 commitdate=$(git show --format=%ci HEAD | head -n 1 | cut - -b1-10 | sed -e 's/-0?/./' | sed -e 's/-0?/./')
 
-echo "$commithash" > commit_hash.txt
```

Comment:
I wonder why we were using `commit_hash.txt` here. @chriseth we should still get the right commit hash without it, right?

Reply:
It does not pack ``.git`` below, so it won't know the hash anymore. I wonder if we should use ``git archive`` here to create an archive from the git checkout and then append the dependencies into that archive...